### ISSUE TITLE
Resize on mobile

### ISSUE THREAD

**Issue by nhnpro** [agargaro/three.ez#23]
I think the scene viewport resize not work on mobile browser. 
- portrait mode: it have small gap above the scene
- when switch to landscape : it not resize

I’m trying to fix it…

![IMG_2743](https://github.com/agargaro/three.ez/assets/18005805/81a7c7af-617c-408c-9849-fd5a038a3ddc)
![IMG_2742](https://github.com/agargaro/three.ez/assets/18005805/b3eb3a3e-9538-4db0-853f-1f181f00b0cc)

**Comment by agargaro** [comment]
Hi, can you provide an example on codepen? 

Are you using a `PerspectiveCameraAuto` or `OrthographicCameraAuto`?

Currently the documentation is still poor, if you need help with anything, please ask :)

**Comment by nhnpro** [comment]
You can check my demo site here:
https://mml-vite-3js.vercel.app/
My CSS is 100% width. You must test on a mobile device browser (my case is iPhone 14), On the Chrome simulator the bug did not occured.

**Comment by agargaro** [comment]
I created an application on stackblitz where I used the css from your demo and tested it on my iphone 8. 

[Stackblitz issue #23](https://stackblitz.com/edit/three-ez-issue-23?file=style.css)

This seems to be the problem:

```css
body {
  margin: 0;
  display: flex;
  place-items: center;
  min-width: 320px;
  min-height: 100vh;
}
```

<img src='https://github.com/agargaro/three.ez/assets/47436453/9a0aa62a-8ac3-4687-8723-a4b04c55a020' width='400'/>

Commenting on these styles, the behaviour would seem to be correct.

Ps. Nice demo! Have you considered using the `interceptByRaycaster` property on the meshes you don't want to have interaction on, to improve performance?
